fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -299,53 +299,46 @@ protected void setOptional(CreateRoutineLoadStmt stmt) throws UserException {
 
         if (Strings.isNullOrEmpty(stmt.getFormat()) || stmt.getFormat().equals("csv")) {
             jobProperties.put(PROPS_FORMAT, "csv");
-            jobProperties.put(PROPS_STRIP_OUTER_ARRAY, "false");
```

Comment:
When the loaded file format is csv, but the json related parameter is configured to true, does it have any effect on the load?

Reply:
No. If format is csv, the json related parameter will always configured to false in analyze state.